packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -177,15 +189,21 @@ const ClinicalResultBloodPressure = (props) => {
 
   const clinicalResultBloodPressureDisplay = (
     <React.Fragment>
-      <div className={decoratedResultClassnames}>
-        <div className={cx('result-display')}>
-          {decoratedResultDisplay}
-          {isTruncated ? null : !hideAccessoryDisplays && iconGroupDisplayElement}
+      <span
+        role={IsBloodPressureAGroup ? 'group' : undefined}
```

Comment:
Do you happen to know the difference between setting this to undefined vs null? I was trying to poke around for more info about how this is handled in relationship to the aria roles but didn't find much - I've seen it done both ways. In general my understanding is that null is used to define something that is programmatically empty and undefined is meant to say that the reference is not existing. I'm pretty sure it shouldn't matter functionality wise but do we care what the role should be associated with?

 i.e. when IsBloodPressureAGroup is false do we want this role to be set to null and therefore implied that it is intentionally a defined reference set to 'nothing' or do we want to set it to undefined which implies that there is no existing property to reference?

I'm definitely overcomplicating this but just wanted to know if you had any thoughts?

Reply:
I think both should work the same but I haven't tested thoroughly with role= null. I was checking the code base and on the [data grid component](https://github.com/cerner/terra-clinical/blob/23aec842dc94c704243ddfcaa6a0883946ad9cde/packages/terra-clinical-data-grid/src/DataGrid.jsx#L1248) I saw that they had used undefined for aria-labelledby property. Hence I went ahead with undefined here as well.